packages/editor/src/client/atoms/poll/pollStateIndication.tsx

```diff
@@ -5,17 +5,20 @@ import React from 'react'
 import {useTranslation} from 'react-i18next'
 import {Tooltip, Whisper} from 'rsuite'
 
-import {FullPoll} from '../../api'
 import {dateTimeLocalString} from '../../utility'
 
 interface PollStateIndicationPorps {
-  poll: FullPoll
+  closedAt: string | null | undefined
+  opensAt: string
 }
 
-export function PollStateIndication({poll}: PollStateIndicationPorps) {
+export function PollStateIndication({
```

Comment:
Just some refactoring that happens when reading through code. Boy scout rule

Removed domain model as not needed.